recipes/lcaclassifier/lcaclassifier.patch

```diff
@@ -0,0 +1,100 @@
+diff -u -r a/LCAClassifier/src/LCAClassifier/arbor.py b/LCAClassifier/src/LCAClassifier/arbor.py
```

Comment:
Please remove this patch file. It doesn't add anything valuable as it only adds shebangs to files that end up in `$PREFIX/lib`.